code/lib/cli/package.json

```
@@ -56,6 +56,7 @@
   "dependencies": {
     "@babel/core": "^7.22.0",
     "@babel/preset-env": "^7.22.0",
+    "@babel/types": "^7.22.5",
```

would it make sense to align the version with the other babel packages?
```suggestion
    "@babel/types": "^7.22.0",
```